strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/MavenArtifactCoordinates.java

```diff
@@ -40,17 +40,6 @@
 
     private static final String EXTENSION = "extension";
 
-    private String groupId;
-
-    private String artifactId;
-
-    private String version;
-
-    private String classifier;
-
-    private String extension = "jar";
```

Comment:
I'm sorry, but I don't understand. Why are these removed? What's going on that I'm missing?

Reply:
Coordinates stored at `GenericArtifactCoordinatesEntity#coordinates` in generic way as `Map` for all layouts.